src/sys/uio.rs

```diff
@@ -56,6 +59,38 @@ pub fn pread(fd: RawFd, buf: &mut [u8], offset: off_t) -> Result<usize>{
     Errno::result(res).map(|r| r as usize)
 }
 
+// process_vm_{read,write}v currently doesn't use flags, but we define this
+// type for forwards-compatibility.
+#[cfg(any(target_os = "linux"))]
+bitflags!(
+    pub struct CmaFlags: libc::c_ulong {
+        // The bitflags! macro requires at least one variant. any() evaluates
+        // to false, so this symbol never actually exists.
+        #[cfg(any())]
+        const CMA_DUMMY = 0;
+    }
+);
+
+#[cfg(any(target_os = "linux"))]
+pub fn process_vm_writev(pid: Pid, local_iov: &[IoVec<&[u8]>], remote_iov: &mut [IoVec<&mut [u8]>], flags: CmaFlags) -> Result<usize> {
+    let res = unsafe {
+        libc::process_vm_writev(pid.into(), local_iov.as_ptr() as *const libc::iovec, local_iov.len() as libc::c_ulong,
+                                            remote_iov.as_ptr() as *const libc::iovec, remote_iov.len() as libc::c_ulong, flags.bits())
+    };
+
+    Errno::result(res).map(|r| r as usize)
+}
+
+#[cfg(any(target_os = "linux"))]
+pub fn process_vm_readv(pid: Pid, local_iov: &mut [IoVec<&mut [u8]>], remote_iov: &[IoVec<&[u8]>], flags: CmaFlags) -> Result<usize> {
+    let res = unsafe {
+        libc::process_vm_readv(pid.into(), local_iov.as_ptr() as *const libc::iovec, local_iov.len() as libc::c_ulong,
+                                           remote_iov.as_ptr() as *const libc::iovec, remote_iov.len() as libc::c_ulong, flags.bits())
```

Comment:
Thanks, I'll add a linebreak before local_iov. (Should I be using rustfmt or something?)